libnopegl/src/path.h

```diff
@@ -50,10 +51,19 @@ int ngli_path_close(struct path *s);
 /* Finalize construction: must be called at the end of the construction */
 int ngli_path_finalize(struct path *s);
 
+/*
+ * Initialization is only required if one wants to evaluate the path at a given
```

Comment:
Initialize a path. It is only required [...]

Reply:
Changed.